src/templates/gene.html

```diff
@@ -36,19 +35,61 @@ <h6 class="m-0 font-weight-bold text-primary">Information</h6>
                                 {% for f in fields %}
                                 {% set param = {f: variant[f]} %}
                                 <td>
+                                    <!-- Items with links -->
                                     {% if f == "dn_no" %}
                                     <a href={{ url_for('patient', id=variant[f]) }}>{{ variant[f] }}</a>
                                     {% elif f == "gene" %}
                                     <a href={{ url_for('gene', id=variant[f]) }}>{{ variant[f] }}</a>
                                     {% elif f == "fullgnomen" %}
-                                    <a href={{ url_for('variant', id=variant[f]) }}>{{ variant[f] }}</a>
+                                        {% for key, value in variant["platformDatasets"].items() %}
+                                        {% if key == "HGVS genomic-level nomenclature (fullGNomen)" %}
+                                        <a href={{ url_for('variant', id=value) }}>{{ value }}</a>
+                                        {% endif %}
+                                        {% endfor %}
+                                    {% elif f == "VUS" %}
+                                    <a href={{ url_for('vus', id=variant["_id"]) }}><i class="fas fa-external-link-square-alt fa-2x"></i></a>
+
+                                    <!-- Items without links -->
+                                    {% elif f == "Position" %}
+                                    {{ variant["chromosome"] }}:{{ variant["start"] }}/{{ variant["stop"] }}
+                                    {% elif f == "cdna" %}
+                                        <div style="min-width: 100%; max-width: 200px; word-wrap: break-word">
```

Comment:
Seems this style format/layout is often used. Could this be defined externally?

Reply:
I was wondering about this as well. In an earlier comment I made an issue to have a more in depth look. However I would like to postpone this to a future release. version 1.0 is not live yet and I expect changes anyway due to first-time users. see #14